.gitignore

```diff
@@ -1,2 +1,3 @@
 .npm
 .coverage
+node_modules
```

Comment:
this is required ? in the package there are a .npm folder, no node_modules from what I see